### PR TITLE
Enhancement/moderator abilities

### PR DESCRIPTION
**Pull request by JoeBrotzer**

### CODE REVIEW

File: stream/views/management.py [hunk 1]
```diff
@@ -100,3 +131,17 @@ def get(self, request, *args, **kwargs):
             data,
             content_type="application/json",
             safe=False)
+
+
+class StatusView(generic.View):
+    def post(self, request, *args, **kwargs):
+        if not request.is_ajax():
+            raise Http404
+        data = ""
```
Comment (rendave) [inline stream/views/management.py]: what is the use of the variable data here?

File: stream/views/management.py [hunk 2]
```diff
@@ -29,6 +34,30 @@ def get_success_url(self):
...
 
 
+class ModeratorView(generic.View):
+    def post(self, request, *args, **kwargs):
+        cond = models.Moderator.objects.filter(
+            owner__username=request.POST['user']).exists()
+        if cond is True:
+            moderator = models.Moderator.objects.get(
+                owner__username=request.POST['user'])
+            print(request.POST['value'])
```
Comment (rendave) [inline stream/views/management.py]: there is a print here

File: stream/templates/stream/lobby/detail_view.html [hunk 3]
```diff
@@ -123,5 +128,10 @@ <h3>You are not logged in...</h3>
 <!-- end of modals -->
 {% endblock content %}
 {% block additionalJS %}
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (JoeBrotzer) [reply stream/templates/stream/lobby/detail_view.html]: Will clear

File: stream/views/management.py (hunk 1 above)
Comment (JoeBrotzer) [reply stream/views/management.py]: Nothing, na carry over ra siguro when I reused some code lol

File: stream/views/management.py (hunk 2 above)
Comment (JoeBrotzer) [reply stream/views/management.py]: Will clear

**Pull request merged**